pkg/printers/tableprinter.go

```diff
@@ -187,8 +195,11 @@ func printTable(table *metav1beta1.Table, output io.Writer, options PrintOptions
 // namespace column. It will fill empty columns with nil (if the object
 // does not expose metadata). It returns an error if the table cannot
 // be decorated.
-func decorateTable(table *metav1beta1.Table, options PrintOptions) error {
+func decorateTable(eventType string, table *metav1beta1.Table, options PrintOptions) error {
```

Comment:
Yuck

Reply:
Can this be a different method `decorateTableWithEventType`?

Reply:
refactored to an `addColumns(position, table, headers, valueFuncs)` method and called from the single place we plumb eventType. Can clean up `decorateTable` to use that as well in a follow up (prepends namespace column, appends label columns)